schema/artist/index.js

```diff
@@ -88,6 +94,38 @@ const ArtistType = new GraphQLObjectType({
               type: GraphQLInt,
               resolve: ({ auction_lots_count }) => auction_lots_count,
             },
+            forsale_artworks_count: {
+              type: GraphQLInt,
+              resolve: ({ forsale_artworks_count }) => forsale_artworks_count,
+            },
+          },
+        }),
+        resolve: (artist) => artist,
+      },
+      overview: {
+        type: new GraphQLObjectType({
+          name: 'ArtistOverview',
+          fields: {
+            bio: {
+              type: GraphQLString,
+              resolve: (artist) => {
+                if (artist.years && artist.hometown && artist.location) {
```

Comment:
Thinking there's gotta be a better way to do this loaded conditional. I thought about creating an array and removing null, undefined, and ' ', value then joining but that didn't really feel right either.